### PR TITLE
Qakbot uninstaller

### PR DESCRIPTION
**Pull request by Neo23x0**
### Summary of the Pull Request

Rule for Quakbot Uninstaller 
https://twitter.com/cyb3rops/status/1697259528709120166?s=20

### Changelog

new: Quakbot uninstaller

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_fbi_qbot_cleanup.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        - Image|endswith: '\QbotUninstall.exe'
+        - Hashes|contains:
+            - 'SHA256=559CAE635F0D870652B9482EF436B31D4BB1A5A0F51750836F328D749291D0B6'
+            - 'SHA256=855EB5481F77DDE5AD8FA6E9D953D4AEBC280DDDF9461144B16ED62817CC5071'
+            - 'SHA256=FAB408536AA37C4ABC8BE97AB9C1F86CB33B63923D423FDC2859EB9D63FA8EA0'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_fbi_qbot_cleanup.yml:22]:
```suggestion
            - 'SHA256=FAB408536AA37C4ABC8BE97AB9C1F86CB33B63923D423FDC2859EB9D63FA8EA0'
            - 'IMPHASH=E772C815072311D6FB8C3390743E6BE5
```

File: rules/windows/process_creation/proc_creation_win_fbi_qbot_cleanup.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        - sha256:
+            - '559cae635f0d870652b9482ef436b31d4bb1a5a0f51750836f328d749291d0b6'
+            - '855eb5481f77dde5ad8fa6e9d953d4aebc280dddf9461144b16ed62817cc5071'
+            - 'fab408536aa37c4abc8be97ab9c1f86cb33b63923d423fdc2859eb9d63fa8ea0'
+        - imphash: 'e772c815072311d6fb8c3390743e6be5'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_fbi_qbot_cleanup.yml:27]:
```suggestion
```

**Comment by nasbench** [comment]
These should also be moved to ET folder. I will do that

File: rules/windows/process_creation/proc_creation_win_fbi_qbot_cleanup.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        - Image|endswith: '\QbotUninstall.exe'
+        - Hashes|contains:
+            - 'SHA256=559CAE635F0D870652B9482EF436B31D4BB1A5A0F51750836F328D749291D0B6'
+            - 'SHA256=855EB5481F77DDE5AD8FA6E9D953D4AEBC280DDDF9461144B16ED62817CC5071'
+            - 'SHA256=FAB408536AA37C4ABC8BE97AB9C1F86CB33B63923D423FDC2859EB9D63FA8EA0'
+            - 'IMPHASH=E772C815072311D6FB8C3390743E6BE5
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_fbi_qbot_cleanup.yml:23]:
```suggestion
            - 'IMPHASH=E772C815072311D6FB8C3390743E6BE5'
```

**Review by nasbench — APPROVED** [review]

**Comment by cybevner** [comment]
Hello.

Shouldn't the hash "7cdee5a5a583eacf24b1f142413aabb4e556ccf4ef3a4764ad084c1526cc90e117" be added as well? It appears in the CISA mitigations section.
"For situational awareness, the following SHA-256 hash is associated with FBI's QakBot uninstaller: 7cdee5a5a583eacf24b1f142413aabb4e556ccf4ef3a4764ad084c1526cc90e117"

Regards

**Pull request merged**